packages/components/src/InfoButton/index.story.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs } from '@storybook/addon-knobs/react'
+
+import InfoButton from '.'
+
+const stories = storiesOf('InfoButton', module)
+stories.addDecorator(withKnobs)
+
+class InfoButtonSample extends React.Component {
+    constructor (props) {
+        super(props)
+    }
+
+    render () {
+        return (
+            <InfoButton/>
```

Comment:
Bu component'in `onClick` prop'u almasi zorunlu oldugu belirtilmis flowType ile ancak burada onClick prop'u almamis bu component. Yani hem calismayacak hem de StoryBook icin hatali bir gosterim.

```<InfoButton onClick={this.someClickFunction} />```

olarak degistirebiliriz.